LICENSE.md

```diff
@@ -1,3 +1,8 @@
+---
+layout: page
+title: CC0 1.0 Universal
+---
+
```

Comment:
Why is this included?  I'm concerned that it may break automatic license checking tools that GitHub has (the one that makes it so the license shows up for the repository, and any other tools that look at GitHub licenses).

On a similar note, it feels like we shouldn't rename this to `.md` since it isn't a markdown file, it is just a text file, and I'm weakly concerned that it may break external tooling that expects a file named `LICENSE` (since that is how GitHub names them).

Is there a problem in Jekyll with just linking directly to a file named `LICENSE`?  It doesn't have to include our site header material/styles I don't think.

Reply:
Yes, there is an issue. Jekyll only renders it if it has a `.md` extension (the file technically _is_ markdown), and if it has a header.

Reply:
> It may break external tooling that expects a file named `LICENSE`

It doesn't. Here's the example GitHub itself provides:https://github.com/hubotio/hubot/blob/master/LICENSE.md

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#determining-the-location-of-your-license